packages/tour-kit/src/components/tour-kit-step.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External Dependencies
+ */
+import classnames from 'classnames';
+/**
+ * Internal Dependencies
+ */
+import type { Config } from '../types';
+
+interface Props {
+	config: Config;
+	currentStepIndex: number;
+	onMinimize: () => void;
+	onDismiss: ( target: string ) => () => void;
+	onNextStepProgression: () => void;
+	onPreviousStepProgression: () => void;
+	onGoToStep: ( stepIndex: number ) => void;
+	setInitialFocusedElement: React.Dispatch< React.SetStateAction< HTMLElement | null > >;
```

Comment:
I think we should avoid redefining all these types here. They are already defined in the `TourStepRendererProps` interface. Maybe import/use that here (also for `TourKitMinimized`)?

Reply:
Ok, I used it like this in the last commit:
```js
interface Props extends TourStepRendererProps {
	config: Config;
}
```

Only thing is that we'll need to also pass the `config.steps` to the component, as `steps` is a `TourStepRendererProps` field.

Reply:
I think that's fine for now. More cleanup through the variant extraction anw. Also considering passing the whole config to the step renderers, but we'll see.